File: Algorithms/1051. Height Checker/javascript/solution.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function (heights) {
    const sorted = [...heights].sort((a, b) => a - b);

    let total = 0;
    for (let i = 0; i < heights.length; i++) {
        if (heights[i] !== sorted[i]) {
            total++;
        }
    }

    return total;
};

File: Algorithms/1051. Height Checker/golang/solution.go
func heightChecker(heights []int) int {
	sorted := make([]int, len(heights))
	copy(sorted, heights)
	sort.Ints(heights)

	var result int
	for i := range heights {
		if heights[i] != sorted[i] {
			result++
		}
	}
	return result
}